### DISCUSSION TITLE
Where can I find the definition for commands=2 in log?

### DISCUSSION THREAD

**Discussion by winston0410** [docker-mailserver/docker-mailserver#3084] [Q&A]
Trying to find the manual and figure out what does the following log mean, but unfortunately no luck:

```
Feb 11 17:53:45 mail postfix/submission/smtpd[1814]: disconnect from unknown[172.17.0.1] ehlo=1 quit=1 commands=2
Feb 11 17:53:47 mail postfix/submission/smtpd[1814]: connect from unknown[172.17.0.1]
```

Is there a list that would represent and help me understand what happened here for `commands=2`?

**Comment by polarathene** [comment]
You can try google `ehlo=1 quit=1 commands=2` or similar, perhaps with Postfix, `smtpd` and `587` / `submission` as additional keywords to search.

This is a log from Postfix software when submitting mail to be sent, likely from a mail client like ThunderBird or RoundCube. I don't know myself what those values are representing, the first two are possibly `1` as `true`, so `commands=2` is probably the more interesting one to look into.

The IP address is most likely from the assigned docker network gateway, it's possible that IPv6 was involved and that was mapped to IPv4 by Docker which is usually not desired. You can avoid the `AAAA` DNS record on your mail domain and then you should get an IP there that isn't a gateway address I think. It might be related to the issue?

**Reply by winston0410** [reply]
Thank you so much for your reply. I have resolved this issue now by trying another configuration, but debugging is difficult because I was sending a email with an opensource software that I didn't have a good knowledge with (Ory Kratos).

Initially I have docker-mailserver handling plaintext only and have this issue. At the end I have used a custom cert for docker mailserver but disable SSL cert verification on Ory Kratos side, which have resolved the issue. Sorry I cannot be more helpful to produce more detail because the issue has been fixed already
